packages/console/src/ds-components/Select/index.tsx

```diff
@@ -42,10 +45,19 @@ function Select<T extends string>({
   isClearable,
   size = 'large',
   dropdownPosition = 'bottom',
+  isSearchEnabled,
 }: Props<T>) {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const [isOpen, setIsOpen] = useState(false);
+  const [searchInputValue, setSearchInputValue] = useState('');
+  const [width, setWidth] = useState(window.innerWidth);
```

Comment:
Where is this `width` state used? I didn't seem to find the usage... Consider removing this?

Reply:
@charIeszhao Sorry, I've removed it. I initially used it to trigger re-renders to update the styles of the input container, now I've stored the styles in a state instead.

Reply:
No problem. But I'm thinking about whether we should use a `ref` other than a `state` for the styles. As they might be changed quite frequently if you scroll or resize the window. Using a state might be too `heavy` as it re-renders the component on state change